### PR TITLE
chore(docs): add `typedoc-json-parser`

### PR DESCRIPTION
**Pull request by RealShadowNova**

### CODE REVIEW

File: .github/workflows/documentation.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        run: |
+          echo "::set-output name=NAME::${GITHUB_REF_NAME}"
+          echo "::set-output name=TYPE::${GITHUB_REF_TYPE}"
+          echo "::set-output name=SHA::${GITHUB_SHA}"
+
+  upload:
+    name: Upload Documentation
+    needs: build
+    strategy:
+      max-parallel: 1
```
Comment (dan-online) [inline .github/workflows/documentation.yml:51]: Why 1?

File: .github/workflows/documentation.yml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+          mkdir -p out/${PACKAGE}
+          mv docs/${PACKAGE}/docs/api.json out/${PACKAGE}/${NAME}.json
+
+      - name: Commit & Push
+        env:
+          PACKAGE: ${{ matrix.package }}
+        run: |
+          cd out
+          git config user.name github-actions[bot]
+          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
```
Comment (dan-online) [inline .github/workflows/documentation.yml:112]: Is this email in ENV somewhere, seems weird to exactly specify the number

Comment (RealShadowNova) [reply .github/workflows/documentation.yml:112]: That's the ID of the bot

File: .github/workflows/documentation.yml (hunk 1 above)
Comment (RealShadowNova) [reply .github/workflows/documentation.yml:51]: 🤷🏻 it's just what others use that's all

Comment (dan-online) [reply .github/workflows/documentation.yml:51]: But we could do it all at once 👀

Comment (RealShadowNova) [reply .github/workflows/documentation.yml:51]: alr

**Review by dan-online — APPROVED** [review]
LGTM

**Pull request merged**